examples/ipynb/webgl_example_1.ipynb

```diff
@@ -27,7 +27,9 @@
     "import vispy.gloo as gloo\n",
     "from vispy import app\n",
     "from vispy.util.transforms import perspective, translate, rotate\n",
-    "app.use_app('ipynb_webgl');"
+    "\n",
+    "# load the vispy bindings for the iPython notebook which enables webGL\n",
```

Comment:
please change all instances of "iPython" to "IPython" -- we're not in the fruit business here :)